src/NUnitFramework/framework/Attributes/AttributeHierarchy.txt

```diff
@@ -5,10 +5,10 @@ NUnitAttribute
 		TestCaseActionAttribute       : 
 		TestSuiteActionAttribute      : 
 	DataAttribute
```

Comment:
Did you intent to remove DataAttribute here? 🙂

Reply:
I just left it there since the attribute isn't removed actually, but marked as obsolete.
On the other hand it sounds reasonable to remove this from `AttributeHierarchy`, because it is just NUnit 2 holdover.

So what do you think?

Reply:
What you've done seems sensible to me. 🙂 We can remove it when we actually remove it from the codebase.